curiouslearning/FeedTheMonsterJS
# Fix CR-151: bangla font changes

# Changes
- prompt-text.ts

# How to test
- Check level 10 & 11 in bangla language

Ref: [CR-151](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-151)


[CR-151]: https://curiouslearning.atlassian.net/browse/CR-151?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
